web/src/components/SubscriptionFormHTML.jsx

```diff
@@ -1,20 +1,30 @@
-import { Button, Input, Space } from "antd";
+import { Typography } from "antd";
+
+import { ThemedButton } from "./ThemedButton";
+import { ThemedInput } from "./ThemedInput";
+
+import "./SubscriptionFormHTML.css";
 
 const SubscriptionFormHTML = () => {
 	return (
-		<div>
+		<div className="subscription-form__container">
 			<form action="/api/subscribe" method="post" id="subscription-form">
-				<Space.Compact>
-					<Input
+				<div className="subscription-form__layout">
```

Comment:
<Space direction="vertical"> Can improve spacing and maintain a clean UI, what do you think?

Reply:
According to the [docs](https://ant.design/components/space#spacecompact), `Space.Compact` is primarily for collapsing borders between multiple connected components. Since we only have a single input here with no connected elements, I'm curious if there's another benefit to using `Space.Compact` that I might be overlooking?